### ISSUE TITLE
Plux breaks all other plugins: AttributeError: module 'plugin.setuptools' has no attribute 'load_plux_entrypoints'

### ISSUE THREAD

**Issue by jdjustdata** [localstack/plux#1]
We recently adopted localstack in some builds and integrations, and install localstack in environments with other package installations. This has been working great, but recently found this error breaks all package installations and builds, and traced it to this package and recent release.

When `plux` is installed in a conda environment it overrides the plugin management for all packages - and not just for localstack.

This is an error that occurs in the ``attrs`` package, for example when `plux` is installed:

```
AttributeError: module 'plugin.setuptools' has no attribute 'load_plux_entrypoints'

ImportError: module 'plugin.setuptools' has no attribute 'load_plux_entrypoints'
```

Our fix is to explicitly uninstall plux after installing localstack. Is there a way to loosen this plugin management?

**Comment by thrau** [comment]
hi Joshua @jdjustdata, the problem was more related to a dependency issue within localstack rather than plux (the plugin system of localstack). in the related localstack PR https://github.com/localstack/localstack/pull/5592 there was a discussion and a user reported it fixed with localstack version 0.14.0.6.

if that's not helping, then it would be great if you could share a minimal working example of your conda setup, or help me better understand what's happening by providing the conda logs (in particular which versions of localstack and, transitively, localstack-ext are being installed)

**Comment by jdjustdata** [comment]
Perfect -- sorry I missed that conversation. We limit our localstack installation to `localstack==0.14.0` because later releases adopt the `TypedDict` typing used in Python 3.8+. All of our products are built in Python 3.7 environments and break if we attempt to promote localstack to a new version:

```
File "/Users/jdaniel/opt/anaconda3/envs/postprocessors/lib/python3.7/site-packages/localstack/cli/localstack.py", line 4, in <module>
    from typing import Dict, Optional, TypedDict
ImportError: cannot import name 'TypedDict' from 'typing'
```

Is there a way to promote `localstack` to 0.14.0.6 without promoting the CLI?

**Comment by thrau** [comment]
a quick workaround that you could try is to also pin `localstack-ext` (transitive dependency of localstack) to `0.14.0.18`

meanwhile i'll fix the localstack CLI python 3.7 compat issue.

**Comment by jdjustdata** [comment]
It looks like these have conflicting dependencies:

```ERROR: Cannot install localstack-ext==0.14.0.18 and localstack==0.14.0 because these package versions have conflicting dependencies.```

from a `requirements.txt` installation with both localstack and the localstack-ext transitive requirement.

For now my workaround is:
- install ``localstack==0.14``
- uninstall ``plux``
- install other dependencies
- install ``plux``
- run ``localstack start -d``

**Comment by thrau** [comment]
i've since released localstack 0.14.0.7 which fixes the compatibility issue with python 3.7

**Comment by jdjustdata** [comment]
That worked perfectly! That was fast. Thanks
